sidebar.css

```diff
@@ -0,0 +1,84 @@
+.docs-container {
+  max-width: 1280px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.docs-content {
+  margin-top: 2rem;
+}
+
+.docs-sidebar {
+  flex: 0 0 200px;
+  margin-right: 5rem;
+  background-color: var(--color-white);
+}
+
+.list-topic {
+  padding: 1rem;
+  font-weight: var(--font-medium);
+  border-bottom: 1px solid var(--color-white);
+  margin-bottom: 0.5rem;
+}
+
+.list-topic a {
```

Comment:
Instead of selecting anchor tag inside list-topic you can create a class named as list-topic-link and pass these properties in it and use that class in your html file.

Reply:
Yeah, that can be done, but I'm curious how would that do any change, let's discuss.